lib/mutant/ast/regexp/transformer/alternative.rb

```diff
@@ -0,0 +1,27 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        class Alternative < self
+          register :regexp_sequence_expression
+
+          class ExpressionToAST < Transformer::ExpressionToAST
+            def call
+              quantify(ast(*children))
+            end
+          end
+
+          class ASTToExpression < Transformer::ASTToExpression
+            def call
+              root = ::Regexp::Expression::Alternative.new(0, 0, 0)
+
```

Comment:
I might suggest removing the blank lines in methods to be consistent with the formatting in https://github.com/mbj/mutant/pull/565/commits/b3b6c22e8868baee9691c75127e3baba833ce8c7#diff-c01564451f86f0ea4fa9bab318f0a5f1R16